modules/administration-guide/pages/networking-requirements.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: CONCEPT
```

Comment:
If this should be a procedure, I could change it and provide the prerequisites and a 1 step procedure to apply the network policy

Reply:
I don't think we need to convert this concept module into a procedure module because most of the content is conceptual and because the procedure is already documented on the linked page, https://docs.openshift.com/container-platform/4.10/networking/network_policy/multitenant-network-policy.html.